.github/workflows/post-merge-verify.yml

```
@@ -0,0 +1,114 @@
+name: post-merge-verify
+# P2-6（ADR-0041）：自动合并的核心安全绳——合并到 main 后自动冒烟，失败自动
+# revert（auto-merge 回滚）。把"事前人审"实质替换为"事后快速回滚"：坏变更在
+# main 的存活时间从"天"压缩到约 10 分钟级（验证+gate）。
+on:
+  push:
+    branches: [main]
+  workflow_call:
+    inputs:
+      smoke:
```

actionlint (1.7.12)

[error] "type" is missing at "smoke" input of workflow_call event

(syntax-check)